packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -251,14 +272,19 @@ export const getStyles = (props: ICheckboxStyleProps): ICheckboxStyles => {
         borderColor: checkboxBorderColorDisabled,
         selectors: {
           [HighContrastSelector]: {
-            borderColor: 'InactiveBorder',

```

Comment:
I am not sure why we had `InactiveBorder` here. I updated to `GrayText` since that's what being used everywhere for disabled state